ItemModCreationBoilerplate/Equipment/EquipmentBase.cs

```diff
@@ -33,11 +31,29 @@ public abstract class EquipmentBase
 
         public static EquipmentIndex Index;
 
-        public abstract void Init(ConfigFile config);
-
         public abstract ItemDisplayRuleDict CreateItemDisplayRules();
 
-        protected void CreateLang()
+        protected abstract void Initialization();
+
+        /// <summary>
+        /// Take care to call base.Init()!
+        /// </summary>
+        public virtual void Init(ConfigFile config)
+        {
+            CreateConfig(config);
+            CreateLang();
+            CreateEquipment();
+            Initialization();
+            Hooks();
+        }
+
+        protected abstract void CreateConfig(ConfigFile config);
```

Comment:
Should be virtual if it's moved to base like this. After all, not all items will have a config. Though this can stay as we pass the config file through, people can just have the method empty I guess.